scope.go

```diff
@@ -85,10 +85,11 @@ type scope struct {
 	histograms      map[string]*histogram
 	histogramsSlice []*histogram
 	timers          map[string]*timer
-	// nb: deliberately skipping timersSlice as we report timers immediately,
-	// no buffering is involved.
+	timersSlice     []*timer
 
 	bucketCache map[uint64]bucketStorage
+	lastReport  time.Time
```

Comment:
initialize value with time.now() on subscope creation?
also may just store an `int64` instead, since we don't care about timezones/monotonicity/etc.

Reply:
Fair point - agreed. Thanks!